backend/lib/routes/terminals.js

```diff
@@ -17,18 +17,34 @@
 'use strict'
```

Comment:
on create/reuse, return name of the terminal resource
on delete, pass name of the terminal resource